torch/onnx/_internal/fx/passes/utils.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
```

Comment:
Should we limit the scope of this utils module by adding explicit guidance here?

Reply:
wdym? could you elaborate

Reply:
Right it was about the module. Sorry for the confusion

Reply:
In addition to docstring, what would be a better file name, `utils.py`, `common.py`, `common_utils.py`, etc?

Reply:
I think we can import passes and use `passes.utils`. `passes.common` sounds great too

Reply:
I'll go with `passes._utils` then, they are not supposed to be used outside lol.